src/templates/wrappers/description.ts

```diff
@@ -11,5 +11,5 @@ import {FieldWrapper} from '../field.wrapper';
   `,
 })
 export class FormlyWrapperDescription extends FieldWrapper {
-  @ViewChild("fieldComponent", {read: ViewContainerRef}) fieldComponent: ViewContainerRef;
+  @ViewChild('fieldComponent', {read: ViewContainerRef}) fieldComponent: ViewContainerRef;
 }
```

Comment:
@mohammedzamakhan should we update codacy if its not consistent. Also codacy support organization so that you can add aitboudad and me.